gensrc/thrift/PlanNodes.thrift

```diff
@@ -364,6 +364,7 @@ struct TOlapScanNode {
   23: optional map<i32, i32> dict_string_id_to_int_ids
   // which columns only be used to filter data in the stage of scan data
   24: optional list<string> unused_output_column_name
+  25: optional bool use_sorted_agg
```

Comment:
Rather than calling this `use_sorted_agg`, maybe call this `preserve_sort_order` (default value = false) because there could be other use cases built on top of a sorted scan.

Reply:
good idea